docs/API.md

```diff
@@ -319,6 +322,218 @@ minioClient.set_bucket_policy('mybucket',
 
 ```
 
+<a name="get_bucket_notification"></a>
+### get_bucket_notification(bucket_name)
+
+Fetch the notifications configuration on a bucket.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``bucket_name``   | _string_  |Name of the bucket.|
+
+__Return Value__
+
+|Param   |Type   |Description   |
+|:---|:---|:---|
+|``Policy``   | _dict_   | If there is no notification configuration, an empty dictionary is returned. Otherwise it has the same structure as the argument to set_bucket_notification   |
+
+__Example__
+
+
+```py
+
+# Get the notifications configuration for a bucket.
+notification = minioClient.get_bucket_notification('mybucket')
+# If no notification is present on the bucket:
+# notification == {}
+
+```
+
+<a name="set_bucket_notification"></a>
+### set_bucket_notification(bucket_name, notification)
+
+Fetch the notifications configuration on a bucket.
```

Comment:
I was still iterating on it. Wanted to see how it looked. Forgot it would update the PR as well. Still working on it.